addons/analytic/static/src/components/analytic_distribution/analytic_distribution.js

```diff
@@ -605,14 +605,25 @@ export class AnalyticDistribution extends Component {
     }
 
     onWindowClick(ev) {
-        //TODO: dragging the search more dialog should not close the popup either
-        const modal = document.querySelector('.modal:not(.o_inactive_modal)');
-        const clickedInSearchMoreDialog = modal && modal.querySelector('.o_list_view') && modal.contains(ev.target);
-        const clickedInKanbanSelectorDialog = modal && modal.querySelector('.o_kanban_view') && modal.contains(ev.target);
+        /*
+        Dropdown should be closed only if all these condition are true:
+            - dropdown is closed
+            - click is outside widget element (widgetRef)
+            - there is no active modal and a list/kanban view
+            - there is no active dialog
+            - there is no popover
+            - click is not targeting document dom element (drag and drop)
```

Comment:
```suggestion
            - dropdown is open
            - click is outside widget element (widgetRef)
            - there is no active modal containing a list/kanban view (search more modal)
            - there is no active dialog (describe this one please. is it not covered by the active modal above?)
            - there is no popover (describe this one please. I don't see this .o_popover class?)
            - click is not targeting document dom element (drag and drop search more modal)
```

Reply:
EDIT: 
- there is no popover (click is not in search modal's search bar menu)

could we `.o_search_bar_menu` instead of `o_popover`